src/auth/email/email.controller.ts

```diff
@@ -1,4 +1,134 @@
-import { Controller } from '@nestjs/common';
+import { Body, Controller, Logger, Post, UseGuards } from '@nestjs/common';
+import {
+  ApiBearerAuth,
+  ApiBody,
+  ApiOperation,
+  ApiResponse,
+  ApiTags,
+} from '@nestjs/swagger';
+import { RequestException } from 'src/common/exception/core/ExceptionBase';
+import { Exceptions } from 'src/common/exception/exceptions';
+import { LoginDto } from './dto/login.dto';
+import { LoginResponseDto } from './dto/login-response.dto';
+import { EmailAuthService } from './email.service';
+import { RegisterDto } from './dto/register.dto';
+import { User } from '@prisma/client';
+import { CurrentUser } from 'src/user/current-user.decorator';
+import { ChangePasswordDto } from './dto/change-password.dto';
+import { ForgotPasswordDto } from './dto/forgot-password.dto';
+import { ResetPasswordDto } from './dto/reset-password.dto';
+import { JwtAuthGuard } from 'src/common/guard/jwt.guard';
 
-@Controller('email')
-export class EmailController {}
+@ApiTags('auth/email')
+@Controller('auth/email')
+export class EmailAuthController {
+  private readonly logger = new Logger(this.constructor.name, {
+    timestamp: true,
+  });
+
+  constructor(private emailAuthService: EmailAuthService) {}
+
+  @Post('login')
+  @ApiOperation({ summary: 'Login user by email and password' })
```

Comment:
One quick suggestion: move these decorators to a separate custom decorator that encompasses all of the applied metadata decorators.

Right now, it seems we have 2/3 decorators, but this can grow and increase clutter. You can do it like this:

```ts
export const EndpointXMetadata = () =>
  applyDecorators(
    UuidParamDecorator(),
    ApiCreatedResponse(RESPONSES_METADATA[200]),
    ApiUnauthorizedResponse(RESPONSES_METADATA[401]),
    ApiForbiddenResponse(RESPONSES_METADATA[403]),
    ApiNotFoundResponse(RESPONSES_METADATA[404]),
    ApiOperation({
      summary: 'Stuff',
      operationId: 'stuff-stuff-stuff',
    }),
    ApiTags('More stuff'),
  );
```